libpod/runtime.go

```diff
@@ -640,6 +640,10 @@ func (r *Runtime) refresh(alivePath string) error {
 		return err
 	}
 
+	if rootless.SkipStorageSetup() {
```

Comment:
This is definitely not safe. This code only runs once per reboot, and the state scrub needs to run.

Reply:
We're still getting a runtime without refresh() having run on it. I don't think that's a good thing. Might be better to error out here and tell them to run another command that doesn't require SkipStorageSetup() to do the one-time refresh

Reply:
I've dropped this part and added a new patch that does that automatically when running as uid != 0, we can take advantage of `podman info` that requires to run in an user namespace